src/test/kotlin/no/nav/familie/ks/sak/kjerne/fagsak/domene/FagsakRepositoryTest.kt

```diff
@@ -0,0 +1,62 @@
+package no.nav.familie.ks.sak.kjerne.fagsak.domene
+
+import no.nav.familie.ks.sak.OppslagSpringRunnerTest
+import no.nav.familie.ks.sak.data.Testdata.randomAktør
+import no.nav.familie.ks.sak.kjerne.personident.AktørRepository
+import org.hamcrest.CoreMatchers.nullValue
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.hamcrest.CoreMatchers.`is` as Is
+
+internal class FagsakRepositoryTest : OppslagSpringRunnerTest() {
```

Comment:
Trenger vi en RepositoryTest? Jeg vet det var i familie-tilbake, men kan unngås her. Istedet kan vi skrive FagsakServiceTest som kaller repo/mokker repo

Reply:
Synes egentlig det er greit å ha gitt at vi lager våre egne spørringer mot DB i repository.
Greit å sjekke at disse fungerer som tiltenkt. Mokker vi f.eks i service test så går vi glipp av det.

Reply:
Ja, enig. Vi kan teste egne spørringer i RepositoryTest. Men hvis vi mockker i service test, må vi gjøre ganske mye for å returnere en komplisert objekt.